src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -13,12 +13,96 @@
 
 (def master-service-status-version 1)
 
+;; Default list of allowed histograms/timers
+(def default-metrics-allowed-hists
+  ["compiler"
```

Comment:
Arguably, it seems like we could move the `jruby.` ones over to the `jruby-metrics` service and the `compiler`, `function`, and `puppetdb` ones over to the `puppet-profiler` service.  I'd be inclined to go that way just to keep the lists a little closer to the code paths that register them but could understand the counter argument that it creates more places to go look for all of the defaults and that the list of services within OSS Puppet Server should for the most part be a black box.